sherpa/astro/ui/utils.py

```diff
@@ -13326,6 +13327,12 @@ def sample_flux(self, modelcomponent=None, lo=None, hi=None, id=None,
         -----
         Setting the Xrays parameter to False is currently unsupported.
 
+        The summary output displayed by this routine - giving the
+        median and confidence ranges - is controlled by the standard
+        Sherpa logging instance, and can be hidden by changing the
+        logging to a level greater than "INFO" (e.g. with
+        sherpa.utils.logging.SherpaVerbosity).
```

Comment:
```suggestion
        `sherpa.utils.logging.SherpaVerbosity`).
```

Reply:
I have found the backtick-links to be less than successful in my documentation so far, but I'l try again (maybe the fact it's fully qualified name will help)

Reply:
Yes, it needs to be the full name for sphinx to find it. If you don't want the full name to appear in the docs, you write
`~sherpa.utils.logging.SherpaVerbosity`, which will show up in the html as "SherpaVerbosity" with a link.

Reply:
I use that in the docs/ setup, but I don't really like the extra noise in the docstrings of functions with the sphinx "extras".